src/ontology/cl-edit.owl

```diff
@@ -17661,12 +17661,15 @@ EquivalentClasses(obo:CL_0002374 ObjectIntersectionOf(obo:CL_0000855 ObjectSomeV
 
 # Class: obo:CL_0002375 (Schwann cell precursor)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") obo:IAO_0000115 obo:CL_0002375 "A glioblast cell that develops from a migratory neural crest cell. The SCP is embedded among neurons (axons) with minimal extracellular spaces separating them from nerve cell membranes and has no basal lamina. In rodents SCPs are the only cells in the Schwann cell linage that expresses Cdh19.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:30971890") Annotation(oboInOwl:hasDbXref "PMID:35815410") Annotation(oboInOwl:hasDbXref "doi:/10.1002/dvdy.20209") obo:IAO_0000115 obo:CL_0002375 "A multipotent progenitor cell that develops from a migratory neural crest cell. The SCP is embedded among axons, with minimal extracellular space separating them from nerve cell membranes. This cell lacks a basal lamina, which distinguishes it from more mature Schwann cells. In rodents, cadherin-19 (Cdh19) serves as a specific marker for this developmental stage.")
 AnnotationAssertion(terms:contributor obo:CL_0002375 <https://orcid.org/0000-0003-1980-3228>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002375 "2010-09-24T02:06:10Z"^^xsd:dateTime)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002375 "Schwann cell precursor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35815410") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_0002375 "SCP")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35815410") rdfs:comment obo:CL_0002375 "SCPs are capable of differentiating into Schwann cells and other cell types such as melanocytes, neurons, and mesenchymal cells.")
 AnnotationAssertion(rdfs:label obo:CL_0002375 "Schwann cell precursor")
-SubClassOf(obo:CL_0002375 obo:CL_0000030)
+SubClassOf(obo:CL_0002375 obo:CL_0011026)
```

Comment:
There is the object property [has potential to develop into](http://purl.obolibrary.org/obo/RO_0002387) that might be useful if you want to say that they have potential to develop into a schwann cell.
I have never used this OP (I have just discovered it), so I will leave it to your judgement if you want to use it.